ui/components/app/assets/types.ts

```diff
@@ -56,6 +56,9 @@ export type TokenWithFiatAmount = Token &
   TokenBalanceValues & {
     isStakeable?: boolean;
     title: string;
+  } & {
+    iconGroup?: { avatarValue: string; symbol: string }[];
+    protocolId?: string;
```

Comment:
I don't think it's a good idea to overload an existing type with more optional properties.